src/Sulu/Bundle/WebsiteBundle/Sitemap/SitemapXMLGenerator.php

```diff
@@ -1,53 +0,0 @@
-<?php
-
-/*
- * This file is part of Sulu.
- *
- * (c) MASSIVE ART WebServices GmbH
- *
- * This source file is subject to the MIT license that is bundled
- * with this source code in the file LICENSE.
- */
-
-namespace Sulu\Bundle\WebsiteBundle\Sitemap;
-
-/**
- * Generate the Sitemap XML based on one or several WebspaceSitemaps.
- */
-class SitemapXMLGenerator implements SitemapXMLGeneratorInterface
```

Comment:
I don't think that this can be achieved to easy ... we should talk about that next week.